### PR TITLE
Cleanup and add debug printers a bit

### PR DESCRIPTION
**Pull request by SkySkimmer**
No more having to manually `install_printer Top_printers.ppuni_level` in ocamldebug!

### CODE REVIEW

**Comment by SkySkimmer** [comment]
@herbelin I split commits a bit to make it clearer what's going on.

**Comment by herbelin** [comment]
Great!

**Comment by SkySkimmer** [comment]
@herbelin is that "PR approved" great or just generic great?

**Comment by herbelin** [comment]
The "great" was an answer to your "split of commits to make it clearer what's going on".

As far as I'm concerned, I'm approving the purpose of this PR but knowing that I did not carefully check the details, and in particular did not check whether no printer had been lost inadvertently in the process. So if someone is willing to carefully look at the PR, s/he shall not feel that the careful technical review has already been done.

(Otherwise, @skyskimmer, if you double-checked yourself the PR, that's enough, as far as I'm concerned, to consider the technical details of the PR as reviewed.)

File: dev/top_printers.ml [hunk 1]
```diff
@@ -149,8 +147,8 @@ let genppj f j = let (c,t) = f j in (c ++ str " : " ++ t)
 
 let ppj j = pp (genppj (envpp pr_ljudge_env) j)
 
-let prsubst s = pp (Mod_subst.debug_pr_subst s)
-let prdelta s = pp (Mod_subst.debug_pr_delta s)
+let ppsubst s = pp (Mod_subst.debug_pr_subst s)
+let ppdelta s = pp (Mod_subst.debug_pr_delta s)
```
Comment (Zimmi48) [inline dev/top_printers.ml]: Is `pp` for pretty-printer and `pr` for just printer? And is this your reason for changing a few names? @herbelin seems to having understood it ("uniformization of naming") but I don't.

**Review by Zimmi48 — COMMENTED** [review]
Kudos for the commit splitting.
This PR looks good and can be merged without much risk. If bugs were to be found, they can be fixed easily I think.
My Makefile-foo is pretty much zero so I don't feel like giving the green mark :S

File: dev/top_printers.ml (hunk 1 above)
Comment (SkySkimmer) [reply dev/top_printers.ml]: `pp` seems to be used for `'a -> unit`, `pr` for `'a -> Pp.t`.

Comment (Zimmi48) [reply dev/top_printers.ml]: Oh! Thanks for the info!

**Review by herbelin — APPROVED** [review]
OK to me.

**Pull request merged**